### PR TITLE
connect: install TensorFlow Serving

### LINKED ISSUE #776: connect: install TensorFlow Serving
Install TensorFlow Serving into the Connect Ubuntu 22 image. This can be done with commands similar to:

```dockerfile
RUN echo "deb [arch=amd64] http://storage.googleapis.com/tensorflow-serving-apt stable tensorflow-model-server tensorflow-model-server-universal" > /etc/apt/sources.list.d/tensorflow-serving.list && \
    curl https://storage.googleapis.com/tensorflow-serving-apt/tensorflow-serving.release.pub.gpg | apt-key add -
RUN export DEBIAN_FRONTEND=noninteractive && \
    apt-get update && \
    apt-get install -y \
    tensorflow-model-server-universal \
    && rm -rf /var/lib/apt/lists/*
```

This is installing `tensorflow-model-server-universal` not `tensorflow-model-server`, as the universal binary has basic optimizations, but no platform-specific instructions.

Additional details about TensorFlow Serving installation: https://www.tensorflow.org/tfx/serving/setup

Connect will separately enable TensorFlow Serving; see https://github.com/rstudio/rstudio-docker-products/issues/777.

### PR DESCRIPTION
**Pull request by aronatkins**
Install the universal binary. No configuration modifications.

Rooted on https://github.com/rstudio/rstudio-docker-products/pull/785

Fixes #776

### CODE REVIEW

**Review by christierney — APPROVED** [review]

File: r-session-complete/README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 # Supported tags and respective Dockerfile links
 
-* [`jammy`, `ubuntu2204`, `jammy-2024.04.0`, `ubuntu2204-2024.04.0`](https://github.com/rstudio/rstudio-docker-products/blob/main/r-session-complete/Dockerfile.ubuntu2204)
```
Comment (dbkegley) [inline r-session-complete/README.md:10]: Any idea why we are bumping the workbench and r-session versions here?

Comment (aronatkins) [reply r-session-complete/README.md:10]: This is so bizarre. I just rebased against main. There is no intent to modify these files. I'll adjust.

Comment (aronatkins) [reply r-session-complete/README.md:10]: Oh. That's why. My mistake.
I rebased against main, not dev. Ugh.

Comment (aronatkins) [reply r-session-complete/README.md:10]: Fixing ...

**Review by dbkegley — APPROVED** [review]

**Pull request merged**